src/actions/index.js

```diff
@@ -136,4 +134,28 @@ export const submitApp = (user, form) => (dispatch) => {
   //   .catch((error) => {
   //     dispatch({ type: types.SUBMISSION_FAIL, error });
   //   });
+  form.grad_date = form.grad_date.replace('-', '');
+  form.status = 0;
```

Comment:
Funny part is status isn't being captured by our backend to indicate the status of the application, we need to address this in https://github.com/slohacks/attendee-application-back-end/blob/master/src/models/ApplicationModel.js

Reply:
Eventually you won't need to add the status since we will default it to 0 on submission

Reply:
Kk. Lemme know when the model is updated and I'll remove it ASAP. :)

Reply:
Removed the status variable.